content/how-to/account-management/account-management-faq/index.md

```diff
@@ -30,13 +30,12 @@ Yes, RBAC is available to all Rackspace customers.
 
 {{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
 
-Yes, you can. For instructions, see [Use Role-Based Access Control
-(RBAC)](/support/how-to/managing-role-based-access-control-rbac).
+Yes, you can. For instructions, see [Use Role-Based Access Control (RBAC)](/support/how-to/managing-role-based-access-control-rbac).
```

Comment:
Do any of your other PRs change the file name of the article found at this link? /support/how-to/managing-role-based-access-control-rbac

If so, then this link won't be able to find that article when user clicks it.
Just double check.

Reply:
Update the link [Use Role-Based Access Control (RBAC)](/support/how-to/managing-role-based-access-control-rbac) with the [Use Role-Based Access Control (RBAC)](/support/how-to/use-role-based-access-control-rbac).